### DISCUSSION TITLE
How to open project on local?

### DISCUSSION THREAD

**Discussion by sellimenes** [shadcn-ui/ui#1126] [Q&A]
Hello, I want to enter the open-source world, and since I have a strong affinity for Shadcn, I'd like to contribute to this project. However, I'm facing difficulties when trying to open the project on my local machine. Is there any documentation available that could guide me through the process?

**Comment by SoyDiego** [comment]
> Hello, I want to enter the open-source world, and since I have a strong affinity for Shadcn, I'd like to contribute to this project. However, I'm facing difficulties when trying to open the project on my local machine. Is there any documentation available that could guide me through the process?

You can check this: https://github.com/shadcn-ui/ui/blob/main/CONTRIBUTING.md
